3/T4/index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/style.css" />
+    <title>T4</title>
+</head>
+<body>
+    <h1>Friendly Numbers</h1>
+    <div class="form-container">
+        <div class="inner-container">
+            <p>Write the beginning of the range</p>
+            <input type="number" id="min" class="input form-control">
+        </div>
+        <div class="inner-container">
+            <p>Write the end of the range</p>
+            <input type="number" id="max" class="input form-control">
```

Comment:
I entered first number, clicked `tab` and entered second number. Then I click `enter` on my keyboard - nothing happens. 

When I fill out the form, I assume that by pressing `enter`, the action will be taken.

Reply:
added